spec/lib/evss/common_service_spec.rb

```diff
@@ -32,4 +32,17 @@
       expect(response).to eq true
     end
   end
+
+  describe '#get_rating_info' do
+    context 'with a valid evss response' do
+      it 'returns a rating info response object' do
+        VCR.use_cassette('evss/disability_compensation_form/find_rating_info_pid') do
+          response = subject.get_rating_info
+          expect(response).to be_ok
+          expect(response).to be_an EVSS::DisabilityCompensationForm::RatingInfoResponse
+          expect(response.user_percent_of_disability).to eq nil
```

Comment:
Does the factory user have a nil value for `user_percent_of_disability`?

Not sure you actually need this line since you already specified the response -- it seems like expecting a nil here _could_ lead to false positives if you get a nil for that value for other reasons.

Reply:
Yes, the factory user does not have a rating so this value comes back as null.